main/templates/join_space/step1.html

```diff
@@ -0,0 +1,79 @@
+
+{% extends "base.html" %}
+{% load widget_tweaks %}
+
+{% block title %}Space Membership{% endblock %}
+
+{% block content %}
+
+<div class="col-md-8">
+    <h2>Join as a Member Space</h2>
+
+    <p>Membership of the Hackspace Foundation is available for 'spaces, providing they are run according to the foundation's <a href="{% url 'foundation' path='define.md' %}">member space definition</a></p>
+
+    <p>If you feel your space <i>mostly</i> fits the definition, feel free to apply and appeal to the directors below.</p>
+
+    <p>A space does not need to have a physical premises to be a member.</p>
+    
+    <p>Member Spaces are full, voting members of the Foundation, the voting power of all space members is capped at 80%, with the remaining 20% being Individual Members.</p>
+
+    <p>The membership fee for a Member Space is a minimum of £20 per year, or 2% of their surplus, collected by Direct Debit. Admission as a member space is at the discretion of the Board.</p>
```

Comment:
This should probably say, 'which ever is greatest'.

Reply:
"A minimum of" implies that, methinks?